src/Microsoft.Azure.WebJobs.Extensions.Kafka/Diagnostics/ActivityTags.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Kafka.Diagnostics
```

Comment:
Adding a documentation with linking to the specs link will make this easier for maintenance for future

Reply:
Added the link.